doc/source/serve/deployment.rst

```diff
@@ -212,6 +212,37 @@ Please refer to the Kubernetes documentation for more information.
 .. _`NodePort`: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
 
 
+
+Health Checking
+===============
+By default, each actor making up a Serve deployment is health checked and restarted on failure.
+
+
+.. note::
+
+   The @serve.healthcheck feature is experimental and may be subject to change before the interface is stabilized. If you have any feedback or run into any issues or unexpected behaviors, please file an issue on GitHub.
```

Comment:
Has `@serve.healthcheck` been replaced with  `check_health()`? If so we should remove this reference to it.

Reply:
yes thank you, updating